docs/sources/reference/api/docker_remote_api_v1.17.md

```diff
@@ -124,6 +124,11 @@ Create a container
              "OpenStdin":false,
              "StdinOnce":false,
              "Env":null,
+             "Labels": [
+                     "Vendor=Acme",
+                     "License=GPL",
+                     "Version=1.0"
+             ],
```

Comment:
@thaJeztah Labels "without values" don't make a lot of sense to me. A label with the empty string as its value, sure, but having a distinct NULL value feels messy.